### PR TITLE
Add missing possible error rules

### PR DESCRIPTION
**Pull request by krienow**

### CODE REVIEW

**Comment by jeddy3** [comment]
@krienow Thanks for the PR.

> `"no-duplicate-selectors": true,`

SGTM.

> `"no-descending-specificity": true,`

I think https://github.com/stylelint/stylelint/issues/2489 will need to go in before this rule is included in the recommended config. That issue is labelled as "help wanted" and "beginner friendly", if anyone fancies giving it a go?

> `"no-unknown-animations": true,`

I'm unsure about this rule as it comes [with a pretty big caveat](https://github.com/stylelint/stylelint/blob/master/lib/rules/no-unknown-animations/README.md):

> This rule considers the identifiers of `@keyframes` rules defined _within the same source_ to be known.

I believe it's only applicable to users who use CSS modules and/or a BEM-based methodology, but not applicable to users who define their animations within a separate source. If we don't include this rule in the config, then we should at least document its exclusion. 

@stylelint/core Anyone with an opinion on these?

**Comment by CAYdenberg** [comment]
I'm in agreement about `no-unknown-animations`, unless we can modify the rule to examine included files. I actually think the bug documented in #2489 is a bit of a corner-case, and would be OK with this going either way.

**Comment by jeddy3** [comment]
> I'm in agreement about `no-unknown-animations`

Cool.

> unless we can modify the rule to examine included files.

I don't think that's going to happen in the built-in rule (maybe in a [plugin though](https://github.com/stylelint/stylelint/issues/2363)). It's also easy to rationalise its exclusion and so it should be easy to communicate its exclusion in the documentation.

> I actually think the bug documented in #2489 is a bit of a corner-case

I've actually run into this myself while writing BEM-based code (and modified components). The rule isn't currently turned on in any of the configs, so it's difficult to judge how many users have been exposed to it. It might be best to be safe rather than sorry with this, especially as that issue shouldn't be too difficult to fix.

**Comment by jeddy3** [comment]
Hopefully, a new contributor will get https://github.com/stylelint/stylelint/issues/2489 done for `Hacktoberfest`, then we can merge this PR in.

File: index.js [hunk 1]
```diff
@@ -16,9 +16,12 @@ module.exports = {
     "function-linear-gradient-no-nonstandard-direction": true,
     "keyframe-declaration-no-important": true,
     "media-feature-name-no-unknown": true,
+    "no-descending-specificity": true,
+    "no-duplicate-selectors": true,
     "no-empty-source": true,
     "no-extra-semicolons": true,
     "no-invalid-double-slash-comments": true,
+    "no-unknown-animations": true,
```
Comment (jeddy3) [inline index.js]: As per the discussion, please remove this rule as it only considers animations defined _within the same stylesheet_ as known.

**Pull request merged**